libs/designsystem/src/lib/components/chart/chart.component.ts

```diff
@@ -84,16 +84,17 @@ export class ChartComponent implements OnInit, OnChanges {
   updateProperties() {
     if (this.options.chart) {
       this.options.chart.height = this.height;
-      this.options.chart.description = this.description;
+      this.options.accessibility.description = this.description;
       switch (this.options.chart.type) {
         case ChartType.PIE:
-          this.options.plotOptions.pie.dataLabels.enabled = this.showDataLabels;
+          (this.options.plotOptions.pie
+            .dataLabels as PlotSeriesDataLabelsOptions).enabled = this.showDataLabels;
```

Comment:
Consider this to avoid cast:
```suggestion
          this.options.plotOptions.series.dataLabels = { enabled: this.showDataLabels };
```